lib/pure/math.nim

```diff
@@ -793,6 +823,14 @@ else: # JS
       doAssert -6.5 mod  2.5 == -1.5
       doAssert  6.5 mod -2.5 ==  1.5
       doAssert -6.5 mod -2.5 == -1.5
+  
+  func divmod*(num, denom: int): (int, int) = 
```

Comment:
why does js get different types? this will break code that expects the return type to match the input types..

Reply:
just for the sake of clarity, you are saying that the signature should be `divmod*[T: SomeInteger](num, denom: T): (T, T)` so that code behaves the same way for both the C(++) and js backends. It's not that the return types don't match the param types because they are both `int`? I'm just making sure that the js back-end doesn't do anything weirder then what I am expecting.

Reply:
```nim
var x: (int8, int8)
x = divmod(2'i8, 3'i8)
```

this snippet should work across backends